src/main/java/com/boundary/plugin/sdk/EventSink.java

```diff
@@ -11,14 +11,20 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.boundary.plugin.sdk;
 
 import java.util.List;
 
 public interface EventSink {
 
     public void emit(Event event);
+
     public void emit(final String event);
+
     public int emit(final List<String> event);
+
+    public boolean openRPCConnection();
```

Comment:
Call these openConnection and closeConnection which will make them generic as EventSink is a generic interface